lib/Jsrt/ChakraCore.h

```diff
@@ -163,7 +163,7 @@ JsGetModuleHostInfo(
     _In_ JsModuleHostInfoKind moduleHostInfo,
     _Outptr_result_maybenull_ void** hostInfo);
 
-#ifndef NTBUILD
+#ifdef CHAKRACOREBUILD_
```

Comment:
`CHAKRACOREBUILD_` is a bit of uncommon style (`_` at the end). Does other option work (CHAKRACORE_BUILD, _CHAKRACORE_BUILD)?